examples/templates/aws-ecs-container/main.tf

```diff
@@ -102,9 +102,10 @@ resource "coder_agent" "coder" {
   startup_script         = <<-EOT
     set -e
 
-    # install and start code-server
-    curl -fsSL https://code-server.dev/install.sh | sh -s -- --version 4.8.3
```

Comment:
Was dropping `--version` intentional?

Reply:
No, I'll add it back for now. Dropping `--version` can be a separate PR.